sparql-ns.ttl

```diff
@@ -70,12 +70,12 @@ sparql:lessThat rdf:type sparql:Operator ;
     rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
     .
    
-sparql:greaterThanOrEquals rdf:type sparql:Operator ;
```

Comment:
`greaterThanOrEquals` and `lessThanOrEquals` also make sense IMO.
But I don't mind the change.